brd-ios/breadwallet/src/Redesign/Scenes/KYC/KYCLevelOne/KYCBasicViewController.swift

```diff
@@ -140,6 +140,9 @@ class KYCBasicViewController: BaseTableViewController<KYCCoordinator,
             view.configure(with: Presets.Button.primary)
             view.setup(with: model)
             view.setupCustomMargins(vertical: .large, horizontal: .large)
+            view.snp.makeConstraints { make in
+                make.height.equalTo(ViewSizes.Common.largeButton.rawValue)
```

Comment:
probably this case can be handled without this? maybe like

view.setupCustomMargins(vertical: .zero, horizontal: .large)

Reply:
no it didn't change, I am changing the button height here